### DISCUSSION TITLE
how to support math (LaTeX) in rspress

### DISCUSSION THREAD

**Discussion by whoimicom** [web-infra-dev/rspress#1249] [Q&A] [answered]
怎么才能在repress中编写markdown数学公式,  是否支持这一特性? 如果支持 需要做一些什么配置以及参考文档是否有?

**Comment by Timeless0911** [comment, marked as answer]
You can use [rspress-plugin-katex](https://github.com/linbudu599/rspress-plugins/tree/main/packages/rspress-plugin-katex)
